flight/AIs/user_command.py

```diff
@@ -0,0 +1,82 @@
+import threading
+
+from ..drone.drone_controller import DroneController
+from .. import constants as c
+from ... import flightconfig as f
+
+def input_loop(controller):
+    """Simple command line interface to drone controller.
+
+    exit: lands and terminates program
+    land: land <priority>
+    hover: hover <duration> <priority>
+    takeoff: takeoff <altitude>
+    move: move <FOWARD,BACKWARD,LEFT,RIGHT> <duration> <priority>
+
+    Notes
+    -----
+    Priority is one of "high", "med", or "low".
+    """
+    while True:
```

Comment:
When does this terminate?

Reply:
It shouldn't unless the user wants it to end, so either by ctrl-c or exit

Reply:
Yeah, what Clay wrote are the two conditions that will cause it to terminate. Is there a better way to go about this?